widgetssdk/src/main/java/com/glia/widgets/UiTheme.java

```diff
@@ -1404,202 +1424,252 @@ public String getAppBarTitle() {
         return appBarTitle;
     }
 
+    @Deprecated
```

Comment:
You will probably need to add docs here to explain the alternative for the deprecated function(s).

Reply:
Yes and no.
We don't have documentation for those methods, so they are our internal methods.
For now, we don't have the alternatives, I just mark the methods that should be moved from here.

But I can link this method with new in the next PRs of the RemoteConfiguration after those methods will be done.